src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2390,7 +2390,7 @@ private BoundExpression BindOutDeclarationArgument(DeclarationExpressionSyntax d
             TypeSymbol fieldType = expressionVariableField.GetFieldType(this.FieldsBeingBound);
             return new BoundFieldAccess(declarationExpression,
                                         receiver,
-                                        expressionVariableField, null, LookupResultKind.Viable, fieldType);
+                                        expressionVariableField, null, LookupResultKind.Viable, fieldType, isDeclaration: true);
```

Comment:
>, isDeclaration: true [](start = 105, length = 21)

I would expect similar change in Binder_Deconstruct.cs

Reply:
Fixed